src/azul/plugins/metadata/hca/transform.py

```diff
@@ -319,18 +319,23 @@ def _donor_types(cls) -> FieldTypes:
             'diseases': str,
             'organism_age': str,
             'organism_age_unit': str,
+            'organism_age_value': str,
             'organism_age_range': None,  # Exclude ranged values from translation, prevents problem due to shadow copies
             'donor_count': None  # Exclude this field added by DonorOrganismAggregator from translation
         }
 
     def _donor(self, donor: api.DonorOrganism) -> MutableJSON:
+        # Disallow age unit being a non-empty string and age value being None
+        assert not donor.organism_age_unit or donor.organism_age
```

Comment:
```suggestion
        # Disallow a null/empty age value if age unit is not empty
        assert donor.organism_age or not donor.organism_age_unit
```
A suggestion, this version of the comment and assert reads a little more clear to me.